### DISCUSSION TITLE
Why can't I have javascript functions from my main.js in onclick?

### DISCUSSION THREAD

**Discussion by john01dav** [parcel-bundler/parcel#5197] [General]
I have a javascript function in my main.js file that is referenced via a `<script src=…` tag in a component that is included via posthtml-include (most of the contents of my <head> tag is processed in this way), and then used via `onclick="functionName()"` in another component that is included via posthtml-include, yet Firefox complains in the JS console that this function does not exist when it is clearly defined in my main.js. Thus, I must conclude that despite the reference to it in the `onclick` entry, Parcel is removing the function. What is the solution to this error?

**Comment by Rishab49** [comment]
Can i have your code?
